source/common/grpc/transcoder_input_stream_impl.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "common/buffer/buffer_impl.h"
+
+#include "src/transcoder_input_stream.h"
+
+namespace Envoy {
+namespace Grpc {
+
+class TranscoderInputStreamImpl : public google::grpc::transcoding::TranscoderInputStream {
+public:
+  // Add a buffer to input stream, will consume all buffer from parameter
+  // if the stream is not finished
+  void Move(Buffer::Instance& instance);
+
+  // Mark the buffer is finished
+  void Finish() { finished_ = true; }
```

Comment:
For all methods that are not inherited from external deps, please follow the Envoy convention and start methods with lower case as per the style guide.